pkg/mtu/mtu.go

```diff
@@ -204,3 +213,12 @@ func (c *Configuration) GetDeviceMTU() int {
 
 	return c.standardMTU
 }
+
+var defaultConfig = Configuration{
+	enableRouteMTUForCNIChaining: false,
+}
+
+func (c Configuration) Flags(flags *pflag.FlagSet) {
+	flags.Bool("enable-route-mtu-for-cni-chaining", c.enableRouteMTUForCNIChaining, "Enable route MTU for pod netns when CNI chaining is used")
```

Comment:
I believe for this to work, enableRouteMTUForCNIChaining needs to be a public field. See go docs here https://github.com/cilium/cilium/blob/2311f3d7811c8950b7f5e1e2ac290f8216be0dc3/vendor/github.com/cilium/hive/cell/config.go#L30

Reply:
Ouch, indeed. Let me test it better.